app/src/main/java/bubbleheads/buy_cook/MainActivity.java

```diff
@@ -14,6 +12,7 @@
 public class MainActivity extends AppCompatActivity {
 
     private final RecipeList recipeList = new RecipeList();
+    private Recipe detailedRecipe;
```

Comment:
generally it is a bed idea to keep this value in activity. I've introduced you example of singletone. Try to reuse it here. So there is some other class with static accessor so that you can always keep and set chosen recipe with it